components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -51,65 +51,65 @@
-    <!-- Indicates the addon is enable. -->
+    <!-- Indicates the add-on is enabled. -->
     <string name="mozac_feature_addons_settings_on">On</string>
-    <!-- Indicates the addon is disable. -->
+    <!-- Indicates the add-on is disabled. -->
     <string name="mozac_feature_addons_settings_off">Off</string>
-    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) addons, this string indicates the installed section. -->
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the installed section. -->
     <string name="mozac_feature_addons_installed_section">Installed</string>
-    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) addons, this string indicates the recommended section. -->
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the recommended section. -->
     <string name="mozac_feature_addons_recommended_section">Recommended</string>
-    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) addons, this string indicates the not yet supported section. -->
+    <!-- This is displayed in a page where the user can see the installed and recommended(not installed yet) add-ons, this string indicates the not yet supported section. -->
     <string name="mozac_feature_addons_unsupported_section">Not yet supported</string>
-    <!-- Displays a page with all the details of an addon. -->
+    <!-- Displays a page with all the details of an add-on. -->
     <string name="mozac_feature_addons_details">Details</string>
-    <!-- Displays a page with all the permissions of an addon. -->
+    <!-- Displays a page with all the permissions of an add-on. -->
     <string name="mozac_feature_addons_permissions">Permissions</string>
-    <!-- This is button that remove (uninstall an addon). -->
+    <!-- This is button that remove (uninstall an add-on). -->
     <string name="mozac_feature_addons_remove">Remove</string>
-    <!-- This is the title of a dialog that indicates to the user if wants to install or not an addon. %1$s is the addon name. -->
+    <!-- This is the title of a dialog that asks the users if they to install or not an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_permissions_dialog_title">Add %1$s?</string>
-    <!-- This is the subtitle of a dialog that indicate to the user if wants to install or not an addon, the subtitle list the permissions that this addon requires, the permissions will dynamically listed after the ":". -->
+    <!-- This is the subtitle of a dialog that asks the users if they wants to install or not an add-on, the subtitle list the permissions that this add-on requires, the permissions will dynamically listed after the ":". -->
```

Comment:
nits: 
* if they *want* to install
* the subtitle *lists"
* will dynamically *be* listed